RHEL/7/input/oval/accounts_logon_fail_delay.xml

```diff
@@ -0,0 +1,31 @@
+<def-group>
+  <definition class="compliance" id="accounts_logon_fail_delay" version="1">
+    <metadata>
+      <title>Ensure that FAIL_DELAY is Configured in /etc/login.defs</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>The delay between failed authentication attempts should be
+      set for all users specified in /etc/login.defs</description>
+      <reference source="galford" ref_id="20160530" ref_url="test_attestation"/>
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_accounts_logon_fail_delay" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check FAIL_DELAY in /etc/login.defs" id="test_accounts_logon_fail_delay" version="1">
+    <ind:object object_ref="object_accounts_logon_fail_delay" />
+    <ind:state state_ref="state_accounts_logon_fail_delay" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_accounts_logon_fail_delay" comment="FAIL_DELAY value from /etc/login.defs" version="1">
+    <ind:filepath>/etc/login.defs</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*(?i)FAIL_DELAY(?-i)[\s]+([^#\s]*)</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+  </ind:textfilecontent54_object>
+  <ind:textfilecontent54_state id="state_accounts_logon_fail_delay" version="1">
+    <ind:subexpression operation="equals" var_ref="var_accounts_fail_delay" datatype="int" />
```

Comment:
The ```operation``` specifier here should be ```greater than or equal``` rather than ```equals```. If I read the [requirement correctly](https://cloud.githubusercontent.com/assets/5713754/14188912/c23c78ba-f758-11e5-9ac6-60fed6aef064.png) any value ```4+``` is OK ("5" should be a valid setting). The current form returns fail when "5" is used.